e2e/cypress/integration/plugins/marketplace/marketplace_spec.js

```diff
@@ -209,8 +214,12 @@ describe('Plugin Marketplace', () => {
 
         after(() => {
             // * cleanup installed plugins
-            cy.uninstallPluginById('github');
-            cy.uninstallPluginById('com.mattermost.webex');
+            cy.getAllPlugins().then((response) => {
+                const active = response.body.active;
+                const inactive = response.body.inactive;
+                inactive.forEach((plugin) => cy.uninstallPluginById(plugin.id));
+                active.forEach((plugin) => cy.uninstallPluginById(plugin.id));
+            });
```

Comment:
Since this is done in several places, can we extract this into a function?  Could be named as `function uninstallAllPlugins() {...}`